steps/ingest.py

```diff
@@ -7,7 +7,7 @@
 from pandas import DataFrame
 
 
-def load_file_as_dataframe(file_path: str, file_format: str) -> DataFrame:
+def load_file_as_dataframe(location: str) -> DataFrame:
```

Comment:
minor: shall we change the name to match the example?

Reply:
I think I like the generic name here since we don't want to suggest to the users that we only support CSV. Keeping this generic name is helpful IMO. What do you think?